.github/workflows/release_workflow.yml

```diff
@@ -0,0 +1,104 @@
+name: CAS Release Workflow
+
+# Controls when the action will run.
+# Triggers the workflow on push or pull request events
+on: [push, workflow_dispatch]
```

Comment:
Similarly if this runs on every push to any branch… won't all these docker images clobber eachother?  For example if user A and user B both have their own feature branches and push to those branches.. they will both push docker images to `:latest`

Typically I've seen docker building only happy on a push to `main` or some other protected branch

Reply:
reading more below it looks like it's not pushing for any commit but only a tagged commit, but that could still conflict right? Might be better to have it on merge to `main` or on the creation of a release (see https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release)